pjmedia/src/pjmedia/vid_stream.c

```diff
@@ -585,7 +584,11 @@ static pj_status_t send_rtcp(pjmedia_vid_stream *stream,
     int len, max_len;
     pj_status_t status;
 
-    pj_mutex_lock(stream->rtcp_mutex);
+
+    /* To avoid deadlock with media transport, we use the transport's
+     * group lock.
+     */
+    pj_grp_lock_acquire( stream->transport->grp_lock );
```

Comment:
The transport may not have the group lock, e.g: (older) transport adapter, so check may be needed?